force-app/main/default/lwc/donationHistoryTable/donationHistoryTable.js

```diff
@@ -26,26 +34,28 @@ export default class DonationHistoryTable extends LightningElement {
             this.paymentMethodLabel = data.fields[PAYMENT_FIELD.fieldApiName].label
         };
         this.columns = [
-            { label: RD2_ScheduleVisualizerColumnDate, fieldName: 'date', type: 'date', typeAttributes:{
+            { label: RD2_ScheduleVisualizerColumnDate, fieldName: 'closeDate', type: 'date-local', typeAttributes:{
                 year: "numeric",
-                month: "short",
-                day: "2-digit"
+                month: "numeric",
+                day: "numeric",
             },
             cellAttributes: { alignment: 'right' }},
-            { label: donationHistoryDonorLabel, fieldName: 'donor', type: 'text' },
+            { label: donationHistoryDonorLabel, fieldName: 'name', type: 'text' },
             { label: commonAmount, fieldName: 'amount', type: 'currency', },
             { label: this.paymentMethodLabel, fieldName: 'paymentMethod', type: 'text', },
         ];
     }
-
-    
-    
-    totalNumberOfRows = MOCK_DATA.length;
     
     // eslint-disable-next-line @lwc/lwc/no-async-await
     async connectedCallback() {
-        const data = MOCK_DATA.slice(0, 50);
-        this.data = data;
+        getDonationHistory({contactId: this.contactId})
+        .then(data => {
+            if (data) {
+                this.allData = data;
+                this.data = data.slice(0, this.recordsToLoad);
```

Comment:
See above, this variable gets lifted out of the class body and into a `const`
```suggestion
                this.data = data.slice(0, RECORDS_TO_LOAD);
```

Reply:
changed